manifests/install.pp

```diff
@@ -123,7 +123,7 @@
             fail('manage_package_repo is not supported on Archlinux')
           }
           package { $::grafana::package_name:
-            ensure  => $::grafana::version, # pacman provider doesn't have feature versionable
+            ensure  => $::grafana::package_ensure, # pacman provider doesn't have feature versionable
```

Comment:
`package_ensure` needs to default to `$::grafana::version` not `$::grafana::params::version`.